plugin/uuid.go

```diff
@@ -0,0 +1,28 @@
+package plugin
+
+import (
+	"fmt"
+	"strconv"
+)
+
+const uuidPattern = "^([a-fA-F0-9]{8}-" +
+	"[a-fA-F0-9]{4}-" +
+	"[%s]" +
+	"[a-fA-F0-9]{3}-" +
```

Comment:
Let's keep the actual UUID grouping together and fuse the two above lines.

Reply:
fixed